lib/osc.py

```diff
@@ -2464,7 +2462,7 @@ def uploadFileToOvf(self, rstUrl=None, method="POST", filePath=None, srcFd=None,
             filePath = urlparse(rstUrl).path
         pass
         fileName = os.path.basename(filePath)
-
+        ## base_path, fileName = os.path.split(filePath)
```

Comment:
remove this line

Reply:
All changes done by rebase, edit, add, commit, fetch.